src/lib/modules/blockchain_connector/index.js

```diff
@@ -411,12 +413,24 @@ class BlockchainConnector {
       'get',
       '/embark-api/blockchain/transactions/:hash',
       (req, res) => {
-        self.getTransaction(req.params.hash, (err, transaction) => {
-          if (err) {
-            self.logger.error(err);
-          }
-          res.send(transaction);
-        });
+        const isRawTxHash = req.query.isRawTxHash && req.query.isRawTxHash === 'true';
+
+        if (isRawTxHash) {
+          self.getTransactionByRawTransactionHash(req.params.hash, (err, transaction) => {
```

Comment:
This is the new API that's supposed to return a transaction object based on a raw transaction hash. Notice that the API itself is a WIP and doesn't fully implement this yet.